samigo/samigo-app/src/webapp/jsf/author/item/fillInTheBlank.jsp

```diff
@@ -114,12 +169,40 @@
     <!-- Extra Credit -->
     <%@ include file="/jsf/author/inc/extraCreditSetting.jspf" %>
 
+    <div class="form-group row">
+        <h:outputLabel value="#{authorMessages.fib_marker}"
+            styleClass="col-md-4 col-lg-2 form-control-label" />
+        <div class="col-md-2 samigo-inline-radio">
+            <h:selectOneRadio value="customMarker" id="customMarker">
+                <f:selectItem itemValue="false" itemLabel="#{authorMessages.fib_marker_default}"
+                    id="default" />
+                <f:selectItem itemValue="true" itemLabel="#{authorMessages.fib_marker_custom}"
+                    id="custom" />
+            </h:selectOneRadio>
+            <h:outputText value="#{authorMessages.fib_note_5}" />
+            <h:message for="customMarker" styleClass="validate" />
+        </div>
+    </div>
+
+    <div class="form-group row" id="customMarkerSettings" style="display: none;">
+        <h:outputLabel value="Custom Markers" styleClass="col-md-4 col-lg-2 form-control-label" />
```

Comment:
Custom Markers is hardcoded

Reply:
I would suggest change value for a property like:

fib_marker_custom_enable=Custom Markers

and in spanish properties:

fib_marker_custom_enable=Delimitadores Personalizados

Reply:
Done, thanks.